tensorflow/core/kernels/mkl_concat_op.cc

```diff
@@ -290,7 +290,16 @@ class MklConcatFwdPrimitive : public MklPrimitive {
       context_.data_mem[i] = *context_.data_mem_shdptr[i];
     }
 
+#ifdef ENABLE_MKLDNN_V1
+    DCHECK_EQ(context_.fwd_primitives.size(),
+              context_.fwd_primitives_args.size());
+    for (size_t i = 0; i < context_.fwd_primitives.size(); ++i) {
+      context_.fwd_primitives.at(i).execute(*context_.fwd_stream,
+                                            context_.fwd_primitives_args.at(i));
+    }
```

Comment:
Please call `execute_primitives` in `mkl_util.h`.

Reply:
thanks for the comment. I will do